src/repository/opamLocal.ml

```diff
@@ -168,9 +168,9 @@ module B = struct
          else
            OpamFilename.mkdir quarantine;
          pull_dir_quiet quarantine url) @@+ function
-    | Not_available _ ->
+    | Not_available (_, msg) ->
       finalise ();
-      Done (OpamRepositoryBackend.Update_err (Failure "rsync failed"))
+      Done (OpamRepositoryBackend.Update_err (Failure ("Not available: " ^ msg)))
```

Comment:
```suggestion
      Done (OpamRepositoryBackend.Update_err (Failure ("rsync error: " ^ msg)))
```
`Not_available` is used when there is an error, we don't know at this stage if it is an availability error or availbility.

Reply:
done